package.json

```diff
@@ -133,69 +134,71 @@
     "number-to-words": "^1.2.4",
     "nunjucks": "^3.2.3",
     "passport": "^0.6.0",
-    "passport-oauth2": "^1.6.1",
-    "qrcode": "^1.5.0",
+    "passport-oauth2": "^1.7.0",
+    "qrcode": "^1.5.1",
     "redis": "^3.1.2",
     "reflect-metadata": "^0.1.13",
     "sqs-consumer": "^5.8.0",
-    "superagent": "^8.0.6",
+    "superagent": "^8.0.9",
     "uk-bank-holidays": "^0.1.0"
   },
   "devDependencies": {
     "@types/bunyan": "^1.8.8",
-    "@types/bunyan-format": "^0.2.4",
+    "@types/bunyan-format": "^0.2.5",
     "@types/compression": "^1.7.2",
     "@types/connect-flash": "0.0.37",
-    "@types/connect-redis": "^0.0.18",
+    "@types/connect-redis": "^0.0.20",
     "@types/csurf": "^1.11.2",
-    "@types/express-request-id": "^1.4.3",
-    "@types/express-session": "^1.17.5",
-    "@types/http-errors": "^1.8.2",
-    "@types/jest": "^27.4.1",
-    "@types/jsonwebtoken": "^9.0.0",
-    "@types/lodash": "^4.14.180",
-    "@types/node": "^14.17.6",
+    "@types/express-request-id": "^3.0.0",
+    "@types/express-session": "^1.17.7",
+    "@types/http-errors": "^2.0.1",
+    "@types/jest": "^29.5.0",
+    "@types/jsonwebtoken": "^9.0.1",
+    "@types/lodash": "^4.14.191",
+    "@types/node": "^18.15.3",
```

Comment:
we're still on node 16 atm. Could bump this but would require [additional changes](https://github.com/ministryofjustice/hmpps-welcome-people-into-prison-ui/pull/352). Probably outside range of this ticket?

Reply:
"@types/node": "^14.17.6",